integration-tests/soak/soak_runner_test.go

```diff
@@ -0,0 +1,163 @@
+package soak_test
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/smartcontractkit/chainlink-starknet/integration-tests/common"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink-env/environment"
+	"github.com/smartcontractkit/chainlink-env/pkg/helm/remotetestrunner"
+	"github.com/smartcontractkit/chainlink-testing-framework/actions"
+	"github.com/smartcontractkit/chainlink-testing-framework/blockchain"
+	"github.com/smartcontractkit/chainlink-testing-framework/logging"
+)
+
+func init() {
+	logging.Init()
+}
+
+var (
+	clImage             = "public.ecr.aws/chainlink/chainlink"
+	clVersion           = "1.9.0"
```

Comment:
mentioned this above, but having hard-coded defaults like this might cause confusion when deploying - compared to ensuring that the CI or user passes it in

Reply:
another reason is that each time a new release is out, we would have to bump the version in the code - otherwise the default would be outdated...